alerter.cpp

```diff
@@ -1,20 +1,28 @@
 #include <iostream>
 #include <assert.h>
 
+#define ALERT_NOT_REQUIRED      (200)
+#define ALERT_REQUIRED          (500)
+
 int alertFailureCount = 0;
 
 int networkAlertStub(float celcius) {

```

Comment:
Stub code is part of production code. Keep test code and production code separate.